### ISSUE TITLE
Quick access: editor picker: Editor group picker doesn't announce group

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#93935]
#93637

Version: 1.44.0-insider (user setup)
Commit: d7d1147d6b764a48b789000c03088ee9afabcd8a
Date: 2020-03-31T08:23:56.446Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

---

In this state with NVDA:

![image](https://user-images.githubusercontent.com/2193314/78029917-2bd3d880-7316-11ea-90ed-206767fba384.png)

It announces: "Quick access dot ts, editor group picker 4 of 5", no mention of which group it is.

**Comment by isidorn** [comment]
Since the `edt` quick pick is about navigation and focus I think reading the group in general makes sense.
If we deicde to change this we should only read the group if there is more than one group open.
